src/util/translations.h

```diff
@@ -43,6 +43,13 @@ class Translations {
             }
             QLocale::setDefault(customLocale);
         }
+#if defined(__APPLE__) && (QT_VERSION < QT_VERSION_CHECK(6, 2, 0))
+        else {
+            // Workaround https://github.com/mixxxdj/mixxx/issues/11195 and
+            // QTBUG-90971 fixed in Qt 6.2
+            QLocale::setDefault(QLocale(QLocale::system().name()));
```

Comment:
Slight optimization since the QString constructor of QLocale warns that its very slow, I don't fully understand the bug though so this should definitely be tested (as should this entire PR). 
```suggestion
            const auto sysLocale = QLocale::system();
            QLocale::setDefault(QLocale(sysLocale.language(), sysLocale.country()));
```